challenges/linked-list/linked-list.js

```diff
@@ -0,0 +1,116 @@
+class CurrentNode {
+  constructor(num) {
+    this.num = num;
+  }
+}
+class LinkList {
+  constructor() {
+    this.head = null;
+    this.length = 0;
+    this.tail = null;
+  }
+
+
+  insert(num) {
+    const newNode = new CurrentNode(num);
+    newNode.next = this.head;
+    this.head = newNode;
+    this.length++;
+  }
+
+  includes(num) {
+    let currentNode = this.head;
+    for(let i = 0; i < this.length; i++) {
+      if(currentNode.num === num) {
+        return true;
+      } else {
+        return false;
+      }
+    }
+  }
+
+  toString() {
+    let string;
+    let currentNode = this.head;
+    for(let i = 0; i < this.length; i++) {
+      string = string + currentNode.num;
+      currentNode = currentNode.next;
+    }
+  }
+
+  append(value) {
+    const newNode = new CurrentNode(value);
+    let currentNode = this.head;
+    while(currentNode.next) {
+      currentNode = currentNode.next;
+    }
+    currentNode = newNode;
+    this.length++;
+  }
+
+  insertBefore(value, newValue) {
+    const newNode = new CurrentNode(newValue);
+    let currentNode = this.head;
+    if(this.head === null) {
+      this.head = newNode;
+      this.length++;
+      return this.head;
+    }
+    let nodeCount = 1;
+    while(currentNode.next !== null && nodeCount < value) {
+      currentNode = currentNode.next;
+      this.length++;
+    }
+    newNode.next = currentNode.next;
+    currentNode.next = newNode;
+    this.length++;
+    return newNode;
+  }
+
+  insertAfter(value, newValue) {
+    const newNode = new CurrentNode(newValue);
+    let currentNode = this.tail;
+    if(this.tail === null) {
+      this.tail = newNode;
+      this.length++;
+      return this.tail;
+    }
+    let nodeCount = 1;
+    while(currentNode.next !== null && nodeCount > value) {
+      currentNode = currentNode.next;
+      this.length++;
+    }
+    newNode.next = currentNode.next;
+    currentNode.next = newNode;
+    this.length++;
+    return newNode;
+  }
+
+  findingKth(k) {
+    let currentNode = this.head;
+    let counter = 0;
+
+    if(k > this.length) {
+      return 'exception';
+    }
+    if(k === this.length) {
+      return 'equal length';
```

Comment:
rather than returning these different strings for different exceptions, I think you just need to return "exception" if the input isn't going to work. I think you read the list of required tests and based your outputs off of those rather than just having those be 5 tests on this method. 

For example, it doesn't need to return "happy place" - instead, you should have a test where you know the input is valid and you should be testing that it returns the node you're expecting. Currently, you don't have any tests actually checking to see if a valid input returns a node.